server/plugin.go

```diff
@@ -460,6 +490,24 @@ func (p *Plugin) syncUsers() {
 			for {
 				newUser, appErr = p.API.CreateUser(newMMUser)
 				if appErr != nil {
+					if appErr.Id == "app.user.save.email_exists.app_error" {
```

Comment:
There will be no need for this check if we get all the users in the above API call.